tests/helpers.py

```diff
@@ -0,0 +1,9 @@
+from deepdiff import DeepDiff
+
+
+def assert_dicts_identical(dict1, dict2):
+    assert len(DeepDiff(dict1, dict2, ignore_order=True)) == 0
+
+
+def assert_lists_contain_same_elements(list1, list2):
+    assert set(list1) == set(list2)
```

Comment:
good candidates for `d3a-interface`

Reply:
Only reason why I didn't port them, was because I thought it was too premature to create a utils module in the d3a-interface for helpers in unit tests. Since you also mentioned that though, you changed my mind, adapting the PR. Thanks!